React/package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "empty-project-template",
+  "name": "compass",
```

Could we possibly name it to something more relevant such as "Location Viewer" etc
```suggestion
  "name": "Location Viewer",
```